migration-engine/connectors/migration-connector/src/steps.rs

```diff
@@ -213,6 +216,7 @@ impl DirectiveLocation {
 #[derive(Debug, Deserialize, Serialize, PartialEq, Clone)]
 #[serde(rename_all = "camelCase", deny_unknown_fields, untagged)]
 pub enum DirectiveType {
+    CustomType { custom_type: String },
```

Comment:
I think we should call it type alias. The term `CustomType` is already used in our context for non standard types offered by different datasources.